### ISSUE TITLE
Error while trying to run demo.

### ISSUE THREAD

**Issue by nikhilbchilwant** [acplt/openAAS#5]
## Current Behaviour
After installation, I am starting server using command : 
`./../../system/sysbin/fb_dbcommands -s localhost:7509/MANAGER -load -f ../../../../addonlibs/administration/FBD/openAAS_Demo_17_10_20.fbd` as mentioned in README.md.
I am getting error:
```
Fehler beim Laden der Datei '../../../../addonlibs/administration/FBD/openAAS_Demo_17_10_20.fbd' in Server 'localhost:7509/MANAGER'.
    Nr. 0x1 (KS_ERR_GENERIC)
```

## Log
Some lines from `openAAS_Demo_17_10_20.log`:
```
ERR Instance /TechUnits/openAAS/AASFolder/Motor_M23.Body/LifeCycle.LifeCycleArchiv couldn't be updated.  Error : KS_ERR_GENERIC
     [ Variable 'identifier' not found ]
     [ Variable 'idH' not found ]
     [ Variable 'idL' not found ]
     [ Variable 'creationtime' not found ]
     [ Variable 'objectstate' not found ]
```
Detailed log here : https://gist.github.com/nikhilbchilwant/0e4bea118668cd937fa3a927ecdc8cd2

## Environment
I am using Ubuntu 16.04 64 bit.

What is going wrong?

**Comment by FlorianPalm** [comment]
Hey,
Have you used the current version of https://github.com/acplt/rte/commit/41942c26343d34031e984d26a52a9c1774e4bf98 ?
The problem could be caused by the fact that you are using a 64 Bit version of Linux.

**Comment by nikhilbchilwant** [comment]
Yes, I did but still no success. It would be great if you could upload a simple YouTube video showing how one should go about running the demo or at least screen shots showing important features of openAAS. 
Recent commits show that this project is evolving rapidly and workshop repository shows that its gaining traction in Industry 4.0 community. Adding more explanation will make it possible for other developers to understand and contribute.

**Comment by ghoddan** [comment]
Hi,
i was at the plugfest on Jan. 29th. Now i wanted to try out the server. but i'm having a similiar issue like described above. 

My error message:
 Fehler beim Laden der Datei '../../../../addonlibs/administration/FBD/openAAS_Demo_17_10_20.fbd' in Server 'localhost:7509/MANAGER'.
    Nr. 0x11 (KS_ERR_BADPATH)

Im running this on a 64Bit Ubuntu 16.04 machine.

Is there a way how to solve this?

Thanks

Ibrahim
